release_notes/AMLS-5546.txt

```diff
@@ -0,0 +1 @@
+ + [AMLS-5546] - 'Would you like to renew your refgistration page and logic'
```

Comment:
Registration, may not be as important as the last considering its just release notes.